### PR TITLE
Add Bitreserve Connect documentation

### PR DESCRIPTION
**Pull request by digobaptista**

### CODE REVIEW

File: _applications.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Scope |  Description
+--------- | -----------
+cards:read | Can view all [cards](#card-object) and their information.
+cards:write | Can create and update any [card](#card-object).
+contacts:read | Can view all [contacts](#contact-object) and their information.
+contacts:write | Can create and update any [contact](#contact-object).
+transactions:read | Can view any [transaction](#transaction-object).
+transactions:write | Can create a [transaction](#transaction-object) from any card to any destination (another card or an external address), cancel and resend transactions.
+user:read | Can view the [user](#user-object) and their information.
```
Comment (fixe) [inline _applications.md]: Missing `phones:read`?

File: _cards.md [hunk 2]
```diff
@@ -137,3 +139,26 @@ currency | | The currency to denominate value stored by the card, represented as
...
+curl https://api.bitreserve.org/v0/me/cards/37e002a7-8508-4268-a18c-7335a6ddf24b \
+  -X PATCH \
+  -H "Authorization: Bearer <token>"
+  -H "Content-Type: application/json" \
+  -d '{ "label": "My Updated Card" }'
+```
+
+### Request
+
+`POST https://api.bitreserve.org/v0/me/cards/:id`
```
Comment (fixe) [inline _cards.md]: `PATCH`.

File: _authentication.md [hunk 3]
```diff
@@ -39,19 +103,65 @@ curl https://api.bitreserve.org/v0/me/tokens \
...
+
+The username should be set as the `token` and password should be set to `X-OAuth-Basic`.
+
+## Basic Authentication
+
+> Simple request using username or email and password:
+
+```bash
+curl https://api.bitreserve.org/v0/me \
+  -H 'X-Bitreserve-OTP: <OTP-Token>' \
```
Comment (fixe) [inline _authentication.md]: Should we mention that OTP isn't required when using a PAT?

Comment (ruimarinho) [inline _authentication.md]: Added mention [here](https://github.com/bitreserve/api/pull/53/files#diff-abc260fb4e7cf8b00d11e36bbf570d59R91).

File: _applications.md (hunk 1 above)
Comment (ruimarinho) [inline _applications.md]: My suggestion is postponing that as we need to document the phone object first.

Comment (fixe) [inline _applications.md]: SGTM.

**Pull request merged**